README.md

```diff
@@ -49,7 +49,7 @@ The balenaDash project includes [wifi-connect](https://github.com/balena-io/wifi
 
 WiFi Connect periodically tests for a functional internet connection, if nothing is found the device sets itself up as a WiFi access point named `balenaDash` that you can join with a mobile device. **Note: due to the fact the built-in WebUI for balenaDash uses port 80, the default captive portal popup screen will not work.**
 
-To use WiFi Connect you need to join the network with a mobile device and ignore the captive portal popup. Ensure that you remain connected to the `balenaDash` network and visit the IP address of the device in a browser on port `8080`. For example `http://<ip of balenaDash device>:8080`. This will allow you to access WiFi Connect, perform a site survey and join a different WiFi network.
+To use WiFi Connect you need to join the network with a mobile device and ignore the captive portal popup. Ensure that you remain connected to the `balenaDash` network and visit the IP address of the device in a browser on port `80`. For example `http://<ip of balenaDash device>`. This will allow you to access WiFi Connect, perform a site survey and join a different WiFi network.
```

Comment:
```suggestion
To use WiFi Connect you need to join the `balenaDash` network and you should see a captive portal popup. If not, ensure that you remain connected to the `balenaDash` network and visit the IP address of the device in a browser on port `80`. For example `http://<ip of balenaDash device>`. This will allow you to access WiFi Connect, perform a site survey and join a different WiFi network.
```